src/components/Button/Button.component.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import "./Button.style.css";
+// import FatArrow from "../ImageComponents/FatArrow.icon";
+
+const Button = ({ children, styleForButton, buttonName }) => {
+  let buttonStyle = "";
+
+  switch (styleForButton) {
+    case "primary":
+      buttonStyle = "white";
+      break;
+
+    case "secondary":
+      buttonStyle = "red";
+
+      break;
+
+    case "seeMore":
+      buttonStyle = "see-more";
+
+      break;
+
+    case "loadMore":
+      buttonStyle = "load-more";
+
+      break;
+
+    case "sortBy":
+      buttonStyle = "sort-by";
+
+      break;
+
+    default:
+      buttonStyle = "regular";
+      break;
+  }
+
+  const classes = `button button__${buttonStyle}`;
+
+  return (
+    <button className={classes}>
+      {buttonName}
+      {children}
```

Comment:
You don't need both of these props that do the same thing, leave only children

Reply:
No, I passed buttonName as a prop, and a FatArrow component as a child, that is why I wrote both.

Reply:
But you are rendering them both in the same place without any difference. For example, if you had something like this
```
<button> <span className="button__icon">{icon}</span> {children}  </button>
```
It would make sense to separate the props, but in your case, both the props will be rendered in the same place without any difference and it wouldn't make sense duplicating it. For example
```
<Button buttonName="Send bid"> <MyIcon/> </Button>
```
and this
```
<Button> Send bid  <MyIcon/> </Button>
```
renders the same thing, but in the second case, you could switch the order of the icon/text and do a lot more, while in the first case you wouldn't be able to do that.